genesyscloud/oauth_client/resource_genesyscloud_oauth_client.go

```diff
@@ -22,6 +22,8 @@ import (
 	"github.com/mypurecloud/platform-client-sdk-go/v154/platformclientv2"
 )
 
+var CacheFile = "/tmp/shared-cache.json"
```

Comment:
The [os.TempDir](https://pkg.go.dev/os#TempDir) function would be useful here. Also better to use `filepath.Join` to make sure the path built is compatible with different operating systems